packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -78,6 +78,69 @@ class MemoryBodyState extends State<MemoryBody> {
     );
   }
 
+  static const String _liveFeed = 'Live Feed';
+  String memorySource = _liveFeed;
+
+  Widget createMenuItem(String name) {
+    final rowChildren = memorySource == name
+        ? [
+            Icon(Icons.check, size: 12),
+            const SizedBox(width: 10),
+            Text(name),
+          ]
+        : [
+            const SizedBox(width: 22),
+            Text(name),
+          ];
+
+    return PopupMenuItem<String>(
+      value: name,
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: rowChildren,
+      ),
+    );
+  }
+
+  Widget _selectMemoryFile() {
+    final List<String> files = _controller.memoryLog.offlineFiles();
+
+    final List<PopupMenuItem<String>> items = [
+      createMenuItem(_liveFeed),
+    ];
+
+    for (var index = 0; index < files.length; index++) {
+      items.add(createMenuItem(files[index]));
+    }
+
+    return PopupMenuButton<String>(
```

Comment:
Use DropdownButton instead so that it is consistent with other pages (profile granularity dropdown for example)

Reply:
Done.